### DISCUSSION TITLE
archives do not open ( Keka )

### DISCUSSION THREAD

**Discussion by vovkaaaA** [aonez/Keka#1561] [General]
hello, please help, a friend threw files compressed into volumes via Keka (he has a mac, and I have Windows) and it doesn't work to open some, I attach a screenshot. Why can't I open them? Winrar, 7-ZIP don't help
![2024-12-13_10-12-09](https://github.com/user-attachments/assets/694f121b-ba27-4884-a364-819e7e96b96a)

**Comment by vovkaaaA** [comment]
UPD: He sees and opens some files ( as in the screenshot ) And some of them don't want to open

**Comment by aonez** [comment]
Is this the same as #1560?

**Comment by aonez** [comment]
You should open the first part, `zip.001`. WinRAR is hiding the `001`, so it is the file with the WinRAR logo. Opening the first part should work for you.

**Reply by vovkaaaA** [reply]
didn't help :(

**Reply by aonez** [reply]
Did you’ve tried with 7-Zip? If the files were sent over internet maybe they got corrupted.

**Reply by aonez** [reply]
You can share them to info@keka.io so I can check their integrity if you want.
